### DISCUSSION TITLE
Public folder use inconsistency

### DISCUSSION THREAD

**Discussion by panieldark** [vercel/next.js#49873] [Help]
### Summary

Hi, tldr: my understanding is that assets, images, fonts, etc. should be available from the base url, (i.e, `public/me.png` should be able to be imported and used like `import myImage from '/me.png'`.

It says here in [public folder docs](https://nextjs.org/docs/getting-started/installation#create-the-public-folder):
> Files inside public directory can then be referenced by your code starting from the base URL (/)

The code snippet in the image docs also shows what seems to be this:
<img width="807" alt="Screenshot 2023-05-16 at 10 40 31 AM" src="https://github.com/vercel/next.js/assets/21699995/c92dc71f-fb2c-4531-9bc0-f6725bba7420">

But scrolling down to the image priority section, there's a relative import usage (which seems scuffed, in practice it would become quite large like `'../../../public/images/logo-navbar.png'`
<img width="806" alt="Screenshot 2023-05-16 at 10 41 12 AM" src="https://github.com/vercel/next.js/assets/21699995/2db431d6-8840-416b-af68-f0632f2f963f">

Could someone give me a definitive answer on this? Should I, or should I not be able to access images as if `public/` was the default asset img location? (and if not, is there a way I can set config so this is the case?)

**Comment by devjiwonchoi** [comment]
Right. In Next.js, `/public` is considered the absolute base URL path and you will be able to import the image as `/public/images/logo-navbar.png`.

The difference from your first example is that you are importing the image from the root `/` path, not from the `/public`.

**Reply by panieldark** [reply]
I see. I think there should be some cleanup in the docs then, as I don't think this is clearly conveyed:

1. Quick wording swap
> Files inside public directory can then be referenced by your code starting from the base URL (/)

This wording (in my interpretation) emphasizes files _inside_ public directory, therefore files should be accessible from `/`. In this example, more correct wording would be "The public directory can then be referenced.."

2. Cleanup usage of relative vs absolute import here (`import profilePic from '/public/me.png`;
![image](https://github.com/vercel/next.js/assets/21699995/c338dff2-df88-4388-b03c-946424618fdf)

**Reply by devjiwonchoi** [reply]
Absolutely agree with your suggestion. You might wanna bring it up to an issue or direct feedback for clearer guidance.

**Comment by devjiwonchoi** [comment]
@panieldark Do you have any other issues or suggestions related to this topic?
